GooglePlayInstant/Editor/PostBuildCommandLineDialog.cs

```diff
@@ -42,23 +42,38 @@ private void Awake()
 
         protected override void Update()
         {
+#if UNITY_2018_3_OR_NEWER
+            // On newer versions of Unity, the AppDomain does not reset after building.
+            // In this case, we run the command when the window is first shown.
```

Comment:
Suggestion for comments:

Starting with Unity 2018.3 the AppDomain is not reset after building an APK.
In this case we run the command when the window is first shown.
Note: some beta versions of 2018.3 reset the AppDomain as in earlier versions of Unity.

Reply:
Done.